### DISCUSSION TITLE
4.9 has undocumented breaking changes

### DISCUSSION THREAD

**Discussion by nicksrandall** [react-icons/react-icons#742] [General]
My app has been using `react-icons/all` for quite some time, and it was removed in 4.9, but there is no mention of this in the release or changelog.

Is there any workaround to get this feature back?

**Comment by kamijin-fanta** [comment]
I am sorry. Due to unusual release procedures, the contents of the v5.0.0 release were included in v4.9.0.

I have added a note about the disruptive change to the release notes. https://github.com/react-icons/react-icons/releases/tag/v4.9.0

Note that it is difficult to revert this functionality. Please move to import by icon set.
